dll/win32/netapi32/local_group.c

```diff
@@ -770,6 +771,9 @@ NetLocalGroupDel(
         goto done;
     }
 
+    /* A successful delete invalidates the handle */
+    AliasHandle = NULL;
+
```

Comment:
Suspiciously wrong, due to the reference counts on the object.
If you want to convince us, prove with a debug trace that when closing the handle on this deleted object, the object becomes instantaneous invalid, whatever the number of handles opened on it...
```suggestion
```